book/src/sprites/load_the_texture.md

```diff
@@ -0,0 +1,66 @@
+# Load The `Texture`
+
+The first part of loading sprites into Amethyst is to read the image into memory. Currently Amethyst supports [`PngFormat`][doc_fmt_png], [`BmpFormat`][doc_fmt_bmp], [`JpgFormat`][doc_fmt_jpg] and [`TgaFormat`][doc_fmt_tga].
+
+The following snippet shows how to load a PNG image:
+
+```rust,no_run,noplaypen
+# extern crate amethyst;
+use amethyst::assets::{AssetStorage, Loader};
+use amethyst::prelude::*;
+use amethyst::renderer::{MaterialTextureSet, PngFormat, Texture, TextureHandle};
+
+pub fn load_texture<N>(name: N, world: &World) -> TextureHandle
+where
+    N: Into<String>,
+{
+    let loader = world.read_resource::<Loader>();
+    loader.load(
+        name,
+        PngFormat,
+        Default::default(),
+        (),
+        &world.read_resource::<AssetStorage<Texture>>(),
+    )
+}
+
+#[derive(Debug)]
+struct ExampleState;
+
+impl<'a, 'b> SimpleState<'a, 'b> for ExampleState {
+    fn on_start(&mut self, data: StateData<GameData>) {
+        let texture_handle = load_texture("texture/sprite_sheet.png", &data.world);
+
+        let texture_id = 0;
+        data.world
+            .write_resource::<MaterialTextureSet>()
+            .insert(texture_id, texture_handle);
+    }
+}
+#
+# fn main() {}
+```
+
+There are two things that may surprise you.
+
+* Firstly, you don't get back the [`Texture`][doc_tex], but a [`TextureHandle`][doc_tex_hd], which is a cloneable reference to the texture.
+
+    When you use [`loader.load(..)`][doc_load] to load an [`Asset`][doc_asset], the method returns immediately with a unique handle for your texture. The actual asset loading is handled asynchronously, so if you attempt to use the texture handle to retrieve the texture, such as with [`world.read_storage::<AssetStorage<Texture>>()`][doc_read_storage][`.get(texture_handle)`][doc_asset_get], you may get a `None` for a few seconds.
```

Comment:
It's now linking to an incorrect api doc for read storage

Reply:
And it should be read_resource

Reply:
ah, fixed again